source/agora/utils/Test.d

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+
+    Various utilities for testing purpose
+
+    Utilities in this module can be used in test code.
+    There are currently multiple testing approaches:
+    - Unittests in the various `agora` module, the most common, cheapest,
+      and a way to do white box testing;
+    - Unittests under `agora.test`: Those unittests rely on the LocalRest
+      library to simulate a network where nodes are thread who communicate
+      via message passing.
+    - Unit integration tests in `${ROOT}/tests/unit/` which are similar to
+      unittests but provide a way to test IO-using code.
+    - System integration tests: those are fully fledged tests that spawns
+      unmodified, real nodes within Docker containers and act as a client.
+
+    Any symbol in this module can be used by any of those method,
+    which is why this module is neither restricted by `package(agora):`
+    nor `version(unittest):`.
+
+    Copyright:
+        Copyright (c) 2019-2020 BOS Platform Foundation Korea
```

Comment:
Not to be Joe, but I'm not sure you can place a copyright year for a time the file didn't exist yet. I think it's based on the file creation date, and not the company creation date?

Reply:
I was thinking that code that is moved should probably keep the right copyright year.
But this isn't moved :D